### PR TITLE
Update team page for Portland 2021

### PR DESCRIPTION
**Pull request by thatdocslady**

### CODE REVIEW

**Review by plaindocs — APPROVED** [review]

File: docs/conf/portland/2021/team.rst [hunk 1]
```diff
@@ -66,20 +66,22 @@ Primary Roles
...
-* **Sponsorship coordinator** - TBD
-* **Writing Day coordinator** - TBD
-* **Volunteer coordinator** - TBD
-* **Emcee** - TBD
-* **Social media coordinator** - TBD
+* **Unconference coordinator** - Jennifer Rondeau
+* **Lightning Talk coordinator** - Rose Williams
+* **Job Fair coordinator** - Mike Jang, Eric Holscher
+* **Welcome Wagon coordinators** - Christy Lutz, Alicia Lewis, Mo Nishiyama
+* **Writing Day coordinator** - Rose Williams, Jennifer Rondeau
```
Comment (thatdocslady) [inline docs/conf/portland/2021/team.rst:76]:
```suggestion
* **Attendee and speaker technical support** - Alex Fornuto
* **Writing Day coordinator** - Rose Williams, Jennifer Rondeau
```

**Review by ericholscher — APPROVED** [review]
Looks good 👍

File: docs/conf/portland/2021/team.rst [hunk 2]
```diff
@@ -66,20 +66,23 @@ Primary Roles
...
+* **Welcome Wagon coordinators** - Christy Lutz, Alicia Lewis, Mo Nishiyama
+* **Attendee and speaker technical support** - Alex Fornuto
+* **Writing Day coordinator** - Rose Williams, Jennifer Rondeau
+* **Emcee** - Andrea Kao, Janine Chan
+* **Q&A moderator** - Jennifer Rondeau, Christy Lutz
+* **Social media coordinator** - Mo Nishiyama
+* **Communication coordinator** - Mikey Ariel
+* **Sponsorship coordinator** - Eric Holscher
+* **Hopin infrastructure coordinator** - Sasha Romijn
+* **Website and email support** - Samuel Wright
```
Comment (ericholscher) [inline docs/conf/portland/2021/team.rst]: Maybe Sasha here too?

Comment (plaindocs) [inline docs/conf/portland/2021/team.rst]:
```suggestion
* **Website and email support** - Samuel Wright, Sasha Romijn
```

**Pull request merged**